components/brave_vpn/resources/panel/components/settings-panel/index.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react'
+import * as S from './style'
+import PanelBox from '../../components/panel-box'
+import { CaratStrongLeftIcon } from 'brave-ui/components/icons'
+import locale from '../../constants/locale'
+
+interface Props {
+  onBack: Function
+}
+
+function SettingsPanel (props: Props) {
+  const handleBackClick = () => props.onBack()
+
+  return (
+    <PanelBox>
+      <S.PanelContent>
+        <S.PanelHeader>
+          <S.BackButton
+            type='button'
+            onClick={handleBackClick}
+          >
+            <i><CaratStrongLeftIcon /></i>
+            <span>{locale.backLabel}</span>
+          </S.BackButton>
+        </S.PanelHeader>
+        <S.List>
+          <li>
+            <a href='#' target='_blank' rel='noopener noreferrer'>
+              {locale.sendFeedback}
+            </a>
+          </li>
+          <li>
+            <a href='#' target='_blank' rel='noopener noreferrer'>
+              About Brave Firewall + VPN
+            </a>
+          </li>
+          <li>
+            <a href='#' target='_blank' rel='noopener noreferrer'>
+              {locale.managePlan}
+            </a>
```

Comment:
Can we put the expected destination Urls in for these before we merge? My concern would be we would forget to put something here if we merge like this as that happens pretty often!

FYI noopener and noreferrer is not required for WebUI, though it won't hurt!

Shouldn't "About Brave Firewall + VPN" have a locale string?

Lastly, are we sure these are going to be links and not buttons that perform internal functions?